questionnaires/templates/questionnaires/form_data.html

```diff
@@ -71,6 +71,15 @@ <h1>
                 </div>
             </div>
         </form>
+          {% if experimental_form_data %}
+            <span class="warning">
+			    <span class="svg-wrapper">
+                    <img src='{% versioned_static "icons/warning.svg" %}' />
+			    </span>
+                In 3 months, the format of form data files in XLSX and CSV will be altered and updated with the
+                implementation of experimental form data.
+		    </span>
+          {% endif %}
```

Comment:
Please format this properly.

Why use `versioned_static` instead of just `static`?